packages/flutter/test/material/text_field_test.dart

```diff
@@ -7398,4 +7398,37 @@ void main() {
     await tester.pump();
     expect(focusNode3.hasPrimaryFocus, isTrue);
   });
+
+  testWidgets('test empty widget when the buildCounter returns null',
+      (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/44909
+
+    final GlobalKey textField1State = GlobalKey();
+    final GlobalKey textField2State = GlobalKey();
```

Comment:
Nitpick: `textField1Key` and `textField2Key`

Reply:
Done.